kadi/events/models.py

```diff
@@ -1466,23 +1481,30 @@ def get_target_attitudes(cls, event, msidset):
 
     @classmethod
     @import_ska
-    def get_one_shot(cls, guide_start, one_shot_msid):
+    def get_one_shot(cls, guide_start, aux_msidset):
         """
         Define one_shot attribute (one-shot attitude update following maneuver)
         """
-        # No acq => guide transition so no one-shot defined.  Use 0.0 as a convenience.
+        # No acq => guide transition so no one-shot defined.  Use -99.0 as a convenience for
+        # the one shot length, -999.0 for the axis values.
         if guide_start is None:
-            return -99.0
-
-        t0 = DateTime(guide_start).secs
-        n_times = len(one_shot_msid.times)
-        i0, i1 = np.searchsorted(one_shot_msid.times, [t0 - 10, t0 + 10])
-        if i0 == 0 or i1 == 0 or i0 == n_times or i1 == n_times:
-            raise ValueError('Unable to find complete data for one-shot attitude value'
-                             ' {} {} {} {} {}'
-                             .format(i0, i1, t0, np.min(one_shot_msid.times),
-                                     np.max(one_shot_msid.times)))
-        return np.max(one_shot_msid.vals[i0:i1])
+            return {'one_shot': -99.0,
+                    'one_shot_roll': -999.0, 'one_shot_pitch': -999.0, 'one_shot_yaw': -999.0}
+        # Save the first post maneuver / post ACQ attitude error sample for each AOATTER axis
+        pm_att_err = {}
+        for ax in [1, 2, 3]:
+            msid = aux_msidset['aoatter{}'.format(ax)]
+            ok = ((msid.times >= DateTime(guide_start).secs)
```

Comment:
I'm fine with going with guide_start + 1.1, but I'm having trouble constructing a situation in my head where you could have a sample with a time greater than guide_start but still not be in GUIDE.  Do you think there are some cases where AOATTER takes more than one sample to update (if so, 1.1 secs seems like it wouldn't help)?  Or if the value of `guide_start` can be wrong (too early) that sounds like a different problem that might need addressing.  Otherwise, the AOATTER values are still from the same telemetry as AOACASEQ and are not interpolated.